view/src/test/java/cmm/apps/esmorga/view/viewmodel/eventdetails/EventDetailsViewModelTest.kt

```diff
@@ -192,7 +202,79 @@ class EventDetailsViewModelTest {
         val joinEventUseCase = mockk<JoinEventUseCase>(relaxed = true)
         coEvery { joinEventUseCase("") } returns EsmorgaResult.success(Unit)
 
-        val sut = EventDetailsViewModel(useCase, userUseCase, joinEventUseCase, "eventId")
+        val leaveEventUseCase = mockk<LeaveEventUseCase>(relaxed = true)
+
```

Comment:
There is a lot of duplicated code in this class, did you tried to move the mockks to a val class?